build/shade/_k-standard-goals.shade

```diff
@@ -226,18 +227,27 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
         var netstandardAppTFM = targetFrameworks.FirstOrDefault(t => t.Equals("netstandardapp1.5", StringComparison.OrdinalIgnoreCase));
         if ((netstandardAppTFM != null || dnxCore50TFM != null) && project.Keys.Contains("testRunner"))
         {
-            DotnetTest(projectFile, Configuration);
+            if (string.IsNullOrEmpty(framework) || framework.Equals(dnxCore50TFM, StringComparison.OrdinalIgnoreCase) || framework.Equals(netstandardAppTFM, StringComparison.OrdinalIgnoreCase))
```

Comment:
You can filter out the `targetFrameworks` to only have the supplied framework and avoid these changes.